### DISCUSSION TITLE
search API gives Failed to retrieve opinions: 502 <!DOCTYPE html>

### DISCUSSION THREAD

**Discussion by haniehm** [freelawproject/courtlistener#4318] [Q&A]
Hi,

As part of an exploratory project, I need to use court listener API to search for title VII cases with their citations: Below parameters where leveraged but I get a 502 after 13 mins and I do get a major part of the response as well. How should I solve the issue?
`# API endpoint
url = 'https://www.courtlistener.com/api/rest/v3/search/'

# Query parameters
params = {
    'q': 'Title VII of the Civil Rights Act of 1964',
    'type': 'o',  # 'o' stands for opinions
    'precedential_status': 'P',
    'order_by': 'citeCount desc',
    'page_size': 100  # Adjust the page size as needed
}`

The error:
Failed to retrieve opinions: 502
<!DOCTYPE html>
<html lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <meta charset="utf-8">
  <meta http-equiv="Content-Language" content="en">
  <meta name="language" content="en_us">
  <meta name="viewport" content="width=device-width,initial-scale=1">
  <link href="/errors_5xx/error-assets/font-awesome.css" rel="stylesheet">

  <title>Yikes, something went wrong â CourtListener.com</title>

  <link rel="stylesheet" href="/errors_5xx/error-assets/bootstrap.css" type="text/css">
  <link rel="stylesheet" href="/errors_5xx/error-assets/override.css" type="text/css" media="screen, projection">
</head>

<body class="">
<div class="container round-bottom">
  <header class="row">
    
    <div class="navbar navbar-default" role="navigation">
      <div class="container-fluid">
        <div class="navbar-header">
          <a class="navbar-brand" href="https://www.courtlistener.com/">
            <svg class="cl-full-logo"
                 viewBox="0 0 635.32 292.37"
                 xmlns:xlink="http://www.w3.org/1999/xlink"
                 xmlns="http://www.w3.org/2000/svg">
              <g id="Layer_2" data-name="Layer 2">
                <g>
                  <rect class="cls-4" x="144.69" y="260.05" width="7.44" height="26.76"/>
                  <rect class="cls-4" x="69.06" y="260.05" width="7.44" height="26.76"/>
                  <rect class="cls-4" x="87.97" y="260.05" width="7.44" height="26.76"/>
                  <rect class="cls-1" x="106.88" y="260.05" width="7.44" height="26.76"/>
                  <rect class="cls-4" x="125.78" y="260.05" width="7.44" height="26.76"/>
                </g>
                <g>
                  <g>
                    <path class="cls-1" d="M63.34,216.92c0-3.14.57-6.24,1.7-9.31,1.13-3.07,2.79-5.81,4.96-8.23,2.17-2.42,4.83-4.37,7.97-5.87,3.14-1.5,6.72-2.25,10.73-2.25,4.73,0,8.83,1.04,12.28,3.12,3.45,2.08,6.03,4.78,7.72,8.12l-7.68,5.29c-.68-1.45-1.5-2.67-2.46-3.66-.97-.99-2.01-1.78-3.12-2.36-1.11-.58-2.27-1-3.48-1.27-1.21-.27-2.39-.4-3.55-.4-2.51,0-4.7.51-6.56,1.52-1.86,1.02-3.41,2.33-4.64,3.95-1.23,1.62-2.16,3.46-2.79,5.51-.63,2.05-.94,4.1-.94,6.12,0,2.27.36,4.46,1.09,6.56.72,2.1,1.75,3.95,3.08,5.54,1.33,1.59,2.92,2.88,4.78,3.84,1.86.97,3.9,1.45,6.12,1.45,1.16,0,2.37-.15,3.62-.43,1.26-.29,2.45-.75,3.59-1.38,1.13-.63,2.19-1.44,3.15-2.43.97-.99,1.74-2.19,2.32-3.59l8.19,4.71c-.82,1.93-1.99,3.64-3.52,5.11-1.52,1.47-3.24,2.72-5.15,3.73-1.91,1.01-3.95,1.79-6.12,2.32-2.17.53-4.3.8-6.38.8-3.67,0-7.03-.77-10.07-2.32-3.04-1.55-5.67-3.56-7.86-6.05-2.2-2.49-3.91-5.31-5.15-8.48-1.23-3.16-1.85-6.39-1.85-9.68Z"/>
                    <path class="cls-1" d="M138.14,243.38c-3.77,0-7.19-.74-10.26-2.21-3.07-1.47-5.7-3.43-7.9-5.87-2.2-2.44-3.9-5.23-5.11-8.37-1.21-3.14-1.81-6.35-1.81-9.64s.64-6.72,1.92-9.86c1.28-3.14,3.04-5.92,5.29-8.33,2.25-2.42,4.92-4.34,8.01-5.76,3.09-1.42,6.45-2.14,10.07-2.14s7.11.76,10.18,2.28c3.07,1.52,5.69,3.52,7.86,5.98,2.17,2.46,3.87,5.27,5.07,8.41,1.21,3.14,1.81,6.33,1.81,9.57,0,3.43-.63,6.7-1.88,9.82-1.26,3.12-3,5.87-5.22,8.26-2.22,2.39-4.88,4.3-7.97,5.73-3.09,1.43-6.45,2.14-10.07,2.14ZM123.21,217.29c0,2.22.34,4.37,1.01,6.45.68,2.08,1.65,3.91,2.94,5.51,1.28,1.59,2.85,2.88,4.71,3.84,1.86.97,3.97,1.45,6.34,1.45s4.55-.49,6.41-1.49c1.86-.99,3.43-2.31,4.71-3.95,1.28-1.64,2.25-3.49,2.9-5.54.65-2.05.98-4.14.98-6.27s-.35-4.37-1.05-6.45c-.7-2.08-1.69-3.91-2.97-5.51-1.28-1.59-2.85-2.86-4.71-3.8-1.86-.94-3.95-1.41-6.27-1.41s-4.57.5-6.45,1.49c-1.88.99-3.46,2.3-4.71,3.91-1.26,1.62-2.21,3.46-2.86,5.51-.65,2.05-.98,4.14-.98,6.27Z"/>
                    <path class="cls-1" d="M192.71,234.54c2.37,0,4.36-.48,5.98-1.45,1.62-.97,2.92-2.22,3.91-3.77.99-1.55,1.69-3.32,2.1-5.33.41-2,.62-4.07.62-6.2v-26.24h9.93v26.24c0,3.53-.42,6.84-1.27,9.93-.85,3.09-2.17,5.8-3.99,8.12s-4.14,4.16-6.99,5.51c-2.85,1.35-6.26,2.03-10.22,2.03s-7.6-.71-10.47-2.14c-2.88-1.42-5.21-3.31-6.99-5.65-1.79-2.34-3.08-5.06-3.88-8.15-.8-3.09-1.2-6.31-1.2-9.64v-26.24h10v26.24c0,2.17.21,4.26.62,6.27.41,2.01,1.11,3.78,2.1,5.33.99,1.55,2.28,2.79,3.88,3.73s3.55,1.41,5.87,1.41Z"/>
                    <path class="cls-1" d="M225.33,243.01v-51.46h22.76c2.37,0,4.55.5,6.56,1.49,2,.99,3.73,2.3,5.18,3.91,1.45,1.62,2.58,3.46,3.41,5.51.82,2.05,1.23,4.12,1.23,6.2,0,3.33-.83,6.37-2.5,9.1-1.67,2.73-3.93,4.7-6.78,5.91l11.74,19.35h-11.23l-10.51-17.25h-9.86v17.25h-10ZM235.33,217h12.54c.92,0,1.78-.22,2.57-.65.8-.43,1.49-1.03,2.07-1.78.58-.75,1.04-1.63,1.38-2.65.34-1.01.51-2.1.51-3.26s-.19-2.32-.58-3.33c-.39-1.01-.91-1.9-1.56-2.65-.65-.75-1.4-1.33-2.25-1.74-.85-.41-1.7-.62-2.57-.62h-12.1v16.67Z"/>
                  </g>
                  <path class="cls-1" d="M310.71,200.33h-16.45v42.69h-9.93v-42.69h-16.45v-8.77h42.83v8.77Z"/>
                  <g>
                    <path class="cls-1" d="M329.91,243.01v-51.46h5.07v46.96h29.72v4.49h-34.79Z"/>
                    <path class="cls-1" d="M368.54,243.01v-51.46h5.07v51.46h-5.07Z"/>
                    <path class="cls-1" d="M412.97,201.2c-1.45-1.64-3.38-2.97-5.8-3.99-2.42-1.01-5.17-1.52-8.26-1.52-4.54,0-7.85.86-9.93,2.57-2.08,1.72-3.12,4.05-3.12,6.99,0,1.55.28,2.82.83,3.81.55.99,1.41,1.85,2.57,2.57,1.16.73,2.65,1.35,4.46,1.88,1.81.53,3.95,1.06,6.41,1.59,2.75.58,5.23,1.23,7.43,1.96,2.2.72,4.07,1.63,5.62,2.72,1.55,1.09,2.73,2.4,3.55,3.95.82,1.55,1.23,3.48,1.23,5.8s-.46,4.42-1.38,6.16c-.92,1.74-2.2,3.19-3.84,4.35-1.64,1.16-3.6,2.02-5.87,2.57-2.27.55-4.76.83-7.46.83-7.97,0-14.86-2.49-20.66-7.46l2.54-4.13c.92.97,2.02,1.88,3.3,2.75,1.28.87,2.71,1.63,4.28,2.28,1.57.65,3.25,1.16,5.04,1.52,1.79.36,3.67.54,5.65.54,4.11,0,7.31-.74,9.6-2.21,2.29-1.47,3.44-3.71,3.44-6.7,0-1.59-.33-2.94-.98-4.02-.65-1.09-1.63-2.03-2.94-2.83-1.3-.8-2.92-1.49-4.86-2.07-1.93-.58-4.18-1.16-6.74-1.74-2.71-.63-5.07-1.28-7.1-1.96-2.03-.68-3.74-1.52-5.15-2.54-1.4-1.01-2.45-2.23-3.15-3.66-.7-1.42-1.05-3.18-1.05-5.25,0-2.37.45-4.48,1.34-6.34.89-1.86,2.16-3.41,3.81-4.64,1.64-1.23,3.58-2.17,5.8-2.83,2.22-.65,4.69-.98,7.39-.98,3.43,0,6.49.52,9.17,1.56,2.68,1.04,5.11,2.5,7.28,4.38l-2.46,4.06Z"/>
                    <path class="cls-1" d="M459.79,196.05h-18.34v46.97h-5.07v-46.97h-18.34v-4.49h41.75v4.49Z"/>
                    <path class="cls-1" d="M498.42,238.52v4.49h-34.57v-51.46h33.92v4.49h-28.85v18.55h25.15v4.28h-25.15v19.64h29.5Z"/>
                    <path class="cls-1" d="M509.43,200.98v42.04h-5.07v-51.46h3.91l34.06,42.83v-42.76h5.07v51.39h-4.42l-33.56-42.04Z"/>
                    <path class="cls-1" d="M591.26,238.52v4.49h-34.57v-51.46h33.92v4.49h-28.85v18.55h25.15v4.28h-25.15v19.64h29.5Z"/>
                    <path class="cls-1" d="M597.2,243.01v-51.46h21.74c2.22,0,4.26.47,6.12,1.41,1.86.94,3.47,2.17,4.82,3.7,1.35,1.52,2.4,3.25,3.15,5.18.75,1.93,1.12,3.89,1.12,5.87s-.28,3.68-.83,5.4c-.56,1.72-1.33,3.26-2.32,4.64-.99,1.38-2.19,2.54-3.59,3.48-1.4.94-2.97,1.58-4.71,1.92l12.61,19.86h-5.73l-12.1-18.99h-15.22v18.99h-5.07ZM602.27,219.53h16.81c1.5,0,2.86-.33,4.1-.98,1.23-.65,2.28-1.53,3.15-2.65.87-1.11,1.55-2.37,2.03-3.77.48-1.4.73-2.87.73-4.42s-.28-3.03-.83-4.46c-.56-1.42-1.3-2.67-2.25-3.73-.94-1.06-2.05-1.91-3.33-2.54-1.28-.63-2.62-.94-4.02-.94h-16.38v23.48Z"/>
                  </g>
                </g>
                <g>
                  <path class="cls-2" d="M193.32,279.1c0,1.01-.2,1.94-.61,2.79-.41.85-.95,1.58-1.63,2.2-.68.62-1.48,1.1-2.39,1.44s-1.89.51-2.93.51h-12.73v-26.99h13c.94,0,1.79.2,2.55.61.76.41,1.41.93,1.94,1.58.53.65.94,1.38,1.24,2.2.29.82.44,1.65.44,2.49,0,1.37-.34,2.62-1.03,3.74-.68,1.13-1.63,1.97-2.85,2.53,1.55.46,2.77,1.31,3.67,2.55.9,1.24,1.35,2.69,1.35,4.33ZM175.69,261.38v9.96h9.27c.66,0,1.27-.14,1.84-.42.57-.28,1.06-.65,1.46-1.1.41-.46.72-.98.95-1.58.23-.6.34-1.22.34-1.88s-.11-1.33-.32-1.94c-.22-.61-.52-1.13-.91-1.58-.39-.44-.86-.8-1.39-1.06-.53-.27-1.12-.4-1.75-.4h-9.5ZM190.66,278.64c0-.66-.11-1.3-.34-1.92-.23-.62-.55-1.16-.97-1.63-.42-.47-.91-.85-1.48-1.14-.57-.29-1.18-.44-1.84-.44h-10.34v10.22h10.07c.68,0,1.32-.14,1.92-.42.59-.28,1.11-.65,1.56-1.12.44-.47.79-1.01,1.05-1.63.25-.62.38-1.26.38-1.92Z"/>
                  <path class="cls-2" d="M200.05,259.07l8.82,14.52,8.89-14.52h2.89l-10.45,16.95v10.03h-2.66v-10.11l-10.41-16.88h2.93Z"/>
                </g>
                <g>
                  <path class="cls-2" d="M238.36,286.05v-26.99h17.71v2.36h-15.05v9.92h12.69v2.2h-12.69v12.5h-2.66Z"/>
                  <path class="cls-2" d="M263.41,286.05v-26.99h11.4c1.17,0,2.24.25,3.21.74.97.49,1.82,1.14,2.53,1.94.71.8,1.26,1.7,1.65,2.72.39,1.01.59,2.04.59,3.08s-.15,1.93-.44,2.83c-.29.9-.7,1.71-1.22,2.43-.52.72-1.15,1.33-1.88,1.82-.74.49-1.56.83-2.47,1.01l6.61,10.41h-3l-6.35-9.96h-7.98v9.96h-2.66ZM266.07,273.74h8.82c.79,0,1.5-.17,2.15-.51s1.2-.8,1.65-1.39c.46-.58.81-1.24,1.06-1.98.25-.73.38-1.51.38-2.32s-.15-1.59-.44-2.34c-.29-.75-.68-1.4-1.18-1.96-.49-.56-1.08-1-1.75-1.33-.67-.33-1.37-.49-2.11-.49h-8.59v12.31Z"/>
                  <path class="cls-2" d="M309.02,283.7v2.36h-18.13v-26.99h17.79v2.36h-15.13v9.73h13.19v2.24h-13.19v10.3h15.47Z"/>
                  <path class="cls-2" d="M334.82,283.7v2.36h-18.13v-26.99h17.79v2.36h-15.13v9.73h13.19v2.24h-13.19v10.3h15.47Z"/>
                </g>
                <path class="cls-2" d="M344.4,286.47c-.68,0-1.26-.22-1.75-.66-.49-.44-.74-1.05-.74-1.84,0-.67.24-1.24.7-1.72.47-.48,1.05-.72,1.74-.72s1.28.24,1.76.71c.49.48.73,1.05.73,1.73,0,.78-.24,1.39-.73,1.83-.49.44-1.06.66-1.71.66Z"/>
                <g>
                  <path class="cls-2" d="M353.01,286.05v-26.99h2.66v24.63h15.58v2.36h-18.24Z"/>
                  <path class="cls-2" d="M386.19,259.07h2.24l11.21,26.99h-2.81l-3.5-8.44h-12.13l-3.46,8.44h-2.85l11.29-26.99ZM392.73,275.52l-5.44-13.34-5.51,13.34h10.95Z"/>
                </g>
                <path class="cls-2" d="M409.07,259.26h2.51l3.8,9.58,3.84-9.58h2.55l-4.71,11.52,5.17,12.2,9.77-23.91h2.89l-11.44,26.99h-2.36l-5.66-13.34-5.7,13.34h-2.32l-11.4-26.99h2.85l9.81,23.91,5.13-12.2-4.71-11.52Z"/>
                <path class="cls-1" d="M108.64.25c-3.36.78-6.07,3.5-6.83,6.86-1.03,4.57,1.38,8.74,5.17,10.47v157.28h7.7V17.59c3.2-1.44,5.43-4.64,5.43-8.37,0-5.79-5.4-10.36-11.47-8.97Z"/>
              </g>
               <a
                 xlink:href="https://free.law/"
                 xlink:show="new">
                  <g id="Layer_4" data-name="Layer 4">
                    <rect class="free-law-link" x="64.43" y="247.23" width="399.41" height="45.14"/>
                  </g>
              </a>
              <g id="Layer_3" data-name="Layer 3">
                <g>
                  <path class="cls-4" d="M208.79,138.68h-47.85v3.95c.02,4.78,2.93,7.71,7.69,7.71,10.42-.02,20.83-.05,31.27-.07.95-.02,1.94-.05,2.86-.3,5.75-1.52,6.81-6.28,6.03-11.29Z"/>
                  <path class="cls-4" d="M219.07,80c-1.73-1.76-3.88-2.63-6.44-2.63-2.77,0-5.03.85-6.79,2.59-.85.83-1.5,1.8-1.92,2.89h-57.83v92.79h7.51s0-85.29,0-85.29h29.05c0,3.35,0,6.7-.05,10.05-.02.65-.12,1.34-.32,1.94-3.44,9.72-6.91,19.42-10.3,29.12-.3.85-.67,1.13-1.52,1.11-3-.05-5.98,0-8.96.02-.99,0-1.39.39-1.39,1.27s.44,1.29,1.5,1.27c2.86,0,5.7-.07,8.57-.09,12.93-.05,25.87-.12,38.82-.16,1.18,0,1.73-.44,1.71-1.29s-.51-1.22-1.66-1.22c-2.47,0-4.92,0-7.37.05-.72.02-1.04-.21-1.32-.88-4.11-9.7-8.22-19.38-12.33-29.08-.14-.3-.21-.62-.21-.95v-11.15h16.12c.39.95.97,1.85,1.71,2.7,1.64,1.85,3.93,2.79,6.88,2.79,2.61,0,4.8-.88,6.54-2.66,1.73-1.76,2.61-3.95,2.61-6.63s-.88-4.78-2.61-6.56ZM195.93,132.35h-19.61c2.93-8.27,5.87-16.56,8.8-24.83h.25c3.51,8.22,7,16.44,10.55,24.83ZM215.26,89.72c-.79.88-1.71,1.32-2.77,1.32s-2.06-.44-2.84-1.34c-.79-.9-1.18-1.99-1.18-3.26,0-1.18.42-2.17,1.2-3,.79-.85,1.73-1.27,2.82-1.27s1.99.46,2.77,1.34c.79.9,1.18,1.94,1.18,3.14s-.39,2.19-1.18,3.07Z"/>
                  <path class="cls-4" d="M62.35,159.33H14.5c0,1.43-.02,2.68,0,3.93,0,4.78,2.93,7.71,7.67,7.71,10.42-.02,20.85-.05,31.27-.07.95,0,1.94-.05,2.86-.3,5.75-1.52,6.81-6.28,6.05-11.27Z"/>
                  <path class="cls-4" d="M17.94,103.65c-.44-1.22-1.13-2.33-2.08-3.3-1.76-1.76-3.9-2.63-6.47-2.63-2.75,0-5.01.85-6.77,2.59s-2.63,3.97-2.63,6.7c0,2.4.81,4.53,2.45,6.4,1.64,1.85,3.93,2.79,6.88,2.79,2.61,0,4.78-.88,6.54-2.63.72-.74,1.27-1.52,1.71-2.4h18.61c0,3.28,0,6.58-.05,9.86,0,.65-.12,1.34-.32,1.94-3.42,9.72-6.88,19.4-10.3,29.12-.3.85-.65,1.11-1.52,1.11-2.98-.05-5.98,0-8.96.02-.97,0-1.39.39-1.39,1.27.02.88.46,1.27,1.52,1.27,2.84-.02,5.7-.07,8.54-.09,12.93-.07,25.89-.12,38.82-.16,1.2-.02,1.73-.44,1.71-1.32,0-.83-.49-1.2-1.66-1.2-2.45,0-4.92-.02-7.37.05-.69,0-1.04-.21-1.32-.88-4.09-9.7-8.22-19.38-12.33-29.08-.12-.3-.21-.65-.21-.97,0-3.63,0-7.3.02-10.95h29.01v64.48h7.51s0-71.98,0-71.98H17.94ZM12.06,110.07c-.81.88-1.71,1.32-2.77,1.32s-2.06-.44-2.84-1.34c-.79-.9-1.18-1.99-1.18-3.26,0-1.15.39-2.17,1.2-3,.79-.83,1.73-1.27,2.82-1.27s1.96.46,2.77,1.34c.79.9,1.18,1.94,1.18,3.14s-.39,2.22-1.18,3.07ZM49.49,152.98h-19.61c2.93-8.27,5.87-16.56,8.8-24.83.07,0,.16,0,.25-.02,3.49,8.22,6.97,16.47,10.55,24.85Z"/>
                  <path class="cls-4" d="M99.67,26.6c-1.73-1.76-3.88-2.63-6.44-2.63-2.75,0-5.01.85-6.77,2.59-1.78,1.73-2.66,3.97-2.66,6.7,0,2.4.83,4.53,2.47,6.4.83.95,1.85,1.66,3.03,2.1v133.88h7.51V41.75c1.06-.44,2.01-1.09,2.86-1.96,1.76-1.76,2.61-3.95,2.61-6.63s-.85-4.78-2.61-6.56ZM95.86,36.33c-.79.88-1.71,1.32-2.75,1.32-1.11,0-2.06-.44-2.84-1.34-.79-.9-1.18-1.99-1.18-3.26,0-1.18.39-2.17,1.18-3,.79-.85,1.73-1.27,2.84-1.27,1.04,0,1.96.46,2.75,1.34.81.9,1.2,1.94,1.2,3.14s-.39,2.19-1.2,3.07Z"/>
                  <path class="cls-4" d="M137.69,48.17c-1.73-1.76-3.9-2.66-6.47-2.66-2.75,0-5.01.88-6.77,2.61-1.76,1.73-2.63,3.95-2.63,6.7,0,2.4.81,4.53,2.45,6.4.83.95,1.85,1.64,3.03,2.1v112.31h7.51V63.32c1.06-.44,2.03-1.11,2.89-1.96,1.73-1.76,2.61-3.97,2.61-6.63s-.88-4.8-2.61-6.56ZM133.88,57.9c-.79.88-1.71,1.32-2.77,1.32s-2.06-.46-2.84-1.34c-.79-.9-1.18-1.99-1.18-3.26,0-1.18.39-2.17,1.2-3,.79-.85,1.73-1.27,2.82-1.27s1.99.44,2.77,1.34c.79.9,1.18,1.94,1.18,3.14s-.39,2.19-1.18,3.07Z"/>
                </g>
              </g>
            </svg>
          </a>
        </div>
      </div>
    </div>
  </header>
  
  <div class="row content">
    <div class="col-sm-3">&nbsp;</div>
    <div class="col-sm-6 text-center">
        <h1 class="text-center">Hmmm, something didn't work, sorry :/</h1>
        <img src="/errors_5xx/error-assets/flip-fail.png" width="200" height=186" alt="A funny image of our old mascot, Flip, with lightning bolts coming out of their head and smashing a computer instead of a dollar sign. On the computer is a fractal image of Flip smashing a computer, which contains Flip smashing a computer, and so forth."/>
      <h3>This usually happens when we are doing maintenance or our server is overloaded.</h3>
      <h3 class="v-offset-below-3">Usually things are fixed within a few minutes.</h3>
        <a href="https://bsky.app/profile/free.law" class="btn btn-primary btn-lg">Check Bluesky for Updates</a>
        <a href="." class="btn btn-default btn-lg"><i class="fa fa-refresh"></i>&nbsp;Refresh Page</a>
    </div>
    <div class="col-sm-3">&nbsp;</div>
  </div>
</div>

<div class="text-center hidden-print" id="social-container">
  <a href="https://free.law/" class="fa-stack fa-lg">
    <i class="fa fa-circle fa-stack-2x gray"></i>
    <i class="fa fa-link fa-stack-1x fa-inverse"></i>
  </a>
  <a href="https://donate.free.law/np/clients/freelawproject/subscribe.jsp?subscription=9" class="fa-stack fa-lg">
    <i class="fa fa-circle fa-stack-2x gray"></i>
    <i class="fa fa-newspaper-o fa-stack-1x fa-inverse"></i>
  </a>
  <a href="https://github.com/freelawproject/courtlistener" rel="noreferrer" class="fa-stack fa-lg">
    <i class="fa fa-circle fa-stack-2x gray"></i>
    <i class="fa fa-github fa-stack-1x fa-inverse"></i>
  </a>
</div>

</body></html>

**Comment by pcd444** [comment]
In my experience this just happens sometimes. I just retry the search and it works most of the time. For me it has never taken 13 min for this to happen but I would just set a timeout to prevent that.

**Comment by mlissner** [comment]
Not sure exactly what's happening here, though we are having some 502's we're working to track down (Kubernetes!!). 

One thing that might help is using v4 of the API, though I suspect this is at a higher level than that.
